modules/ROOT/pages/_partials/task-flex-add-tls-context.adoc

```diff
@@ -41,12 +41,12 @@ Refer to the following cross reference table for the required TLS context config
 | *Keystore* | Required | Required | Not Used | Required
 | *Truststore* | Not Used | Required | Required, unless skip server cert validation is selected | Required to validate upstream certificate
 | *Validate Client certificate* | Not Selected | Selected | Not Used | Not Used
-| *Skip server certificate validation* | Not Used | Not Used | Selected | Not Selected 
+| *Skip server certificate validation* | Not Used | Not Used | Either | Not Selected 
 |===
 
 A parameter that is "Not Used" for a configuration implies its status does not affect configuration.
 
-To simplify your configurations, you can create different TLS Contexts for the different traffic directions of your API Instance. However, you can also use the same TLS Context to support different traffic directions, but you cannot support TLS and mTLS for the same direction using the same context. If you use the same TLS context for different directions, ensure you include the required parameters for both directions.
+To simplify your configurations, you can create different TLS Contexts for the different traffic directions of your API instance. However, you can also use the same TLS Context to support different traffic directions, but you cannot support TLS and mTLS for the same direction using the same context. If you use the same TLS context for different directions, ensure you include the required parameters for both directions.
```

Comment:
In the other PR, context is not capitalized for instances of TLS context. Try to be consistent when you can.